### ISSUE TITLE
Please make roop unleashed 4.0.0 available on repository 

### ISSUE THREAD

**Issue by Michael2Miller** [C0untFloyd/roop-unleashed#684]
Please make roop unleashed 4.0.0 available on repository 
I tried it on kaggle but it still 3.9.0
![IMG_20240513_134853_080](https://github.com/C0untFloyd/roop-unleashed/assets/168263994/c26e0825-83ac-4f70-86e8-ea9599a0deef)

**Comment by Michael2Miller** [comment]
![-5048687387789601840_121](https://github.com/C0untFloyd/roop-unleashed/assets/168263994/7a4acf56-2cd2-437d-9a25-c0105167578b)
It's weird it's not Installing the new requirements  I think the problem is from my side but I don't know how to fix it

**Comment by C0untFloyd** [comment]
It looks like you already had that older roop-unleashed folder and it couldn't be overwritten with the new version. Try deleting all the existing folders and start again new.

**Comment by Michael2Miller** [comment]
Yeah you are right I did that and it's working thank you very much, it was my mistake, you are the best because you always reply faster than any other developer,

**Comment by C0untFloyd** [comment]
Looks like I either have no life or am very good at procrastination 😄 
Probably the latter...
